app/src/main/java/org/mozilla/fenix/library/downloads/DownloadFragment.kt

```diff
@@ -41,9 +42,6 @@ class DownloadFragment : LibraryPageFragment<DownloadItem>(), UserInteractionHan
     private lateinit var downloadStore: DownloadFragmentStore
     private lateinit var downloadView: DownloadView
     private lateinit var downloadInteractor: DownloadInteractor
-    private var undoScope: CoroutineScope? = null
-    private var pendingDownloadDeletionJob: (suspend () -> Unit)? = null
-    private lateinit var downloadsUseCases: DownloadsUseCases
```

Comment:
Should this part of another ticket, as it's not part of the history group?

Reply:
Yes! It is actually part of a [Working undo](https://github.com/mozilla-mobile/fenix/pull/24398) ticket